smartsim/_core/launcher/step/lsfStep.py

```diff
@@ -221,16 +221,16 @@ def _build_exe(self) -> t.List[str]:
         args = self.run_settings.exe_args
         if self.run_settings.mpmd:
             erf_file = self.get_step_file(ending=".mpmd")
-            _ = self._make_mpmd()
+            self._make_mpmd()
```

Comment:
I dislike this change -- explicitly showing that there is a return that we're disregarding is useful internal documentation in code

Reply:
`_make_mpmd()` on lsfStep (unlike alps/slurm/mpi) doesn't return anything. We're not disregarding the result here so it's safe. I would say it's a questionable LSF implementation, though out of the purview of this task to fix.

![image](https://github.com/CrayLabs/SmartSim/assets/3595025/6c6b63c2-cb64-4d4e-b8fa-63392690a6b5)